app/views/triggers/_trigger_fields.html.haml

```diff
@@ -57,7 +57,7 @@
       #events
         = f.association :events, label_method: :name_and_id, label: false
 
-- unless local_assigns[:create_trigger]
+- unless local_assigns[:no_submit_button] # not a part of text component form
```

Comment:
true. I have to use double negation, because the submit button is always there *unless* somethign is set.

I'll see if I can refactor it